Zilon.Core/Zilon.Core/PersonModules/InventoryModule.cs

```diff
@@ -0,0 +1,18 @@
+﻿using Zilon.Core.Props;
+
+namespace Zilon.Core.PersonModules
+{
+    /// <summary>
+    /// Инвентарь персонажа.
+    /// </summary>
+    public sealed class InventoryModule : PropStoreBase, IInventoryModule
+    {
+        public InventoryModule(): base()
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this redundant 'base()' call.](https://app.codacy.com/manual/kreghek/Zilon_Roguelike/pullRequest?prid=5453268)